src/scripts/farming/mutation/mutationTypes/EnigmaMutation.ts

```diff
@@ -58,7 +58,22 @@ class EnigmaMutation extends GrowMutation {
      */
     static getReqs(): BerryType[] {
         SeededRand.seed(+App.game.discord.ID());
-        const berryTypes = Farming.getGeneration(2).concat(Farming.getGeneration(3));
+        // Getting possible Berries
+        // Only Gen 3 and 4 Berries
+        let berryTypes = Farming.getGeneration(2).concat(Farming.getGeneration(3));
+        // Remove parasite Berries
+        berryTypes = berryTypes.filter(berry => {
+            if (berry == BerryType.Occa) {
+                return false;
+            }
+            if (berry == BerryType.Kebia) {
+                return false;
+            }
+            if (berry == BerryType.Colbur) {
+                return false;
+            }
+            return true;
```

Comment:
Could we simplify this to something like this
```suggestion
            return ![BerryType.Occa, BerryType.Kebia, BerryType.Colbur].includes(berry);
```